oap-server/oal-rt/src/main/java/org/apache/skywalking/oal/rt/OALRuntime.java

```diff
@@ -377,10 +376,19 @@ private Class generateDispatcherClass(String scopeName,
              */
             String sourceClassName = SOURCE_PACKAGE + dispatcherContext.getSource();
             SignatureAttribute.ClassSignature dispatcherSignature = new SignatureAttribute.ClassSignature(null, null,
-                // Set interface and its generic params
-                new SignatureAttribute.ClassType[] {
-                    new SignatureAttribute.ClassType(SourceDispatcher.class.getCanonicalName(), new SignatureAttribute.TypeArgument[] {new SignatureAttribute.TypeArgument(new SignatureAttribute.ClassType(sourceClassName))})
-                });
+                                                                                                          // Set interface and its generic params
+                                                                                                          new SignatureAttribute.ClassType[] {
+                                                                                                              new SignatureAttribute.ClassType(
+                                                                                                                  SourceDispatcher.class
+                                                                                                                      .getCanonicalName(),
+                                                                                                                  new SignatureAttribute.TypeArgument[] {
+                                                                                                                      new SignatureAttribute.TypeArgument(
+                                                                                                                          new SignatureAttribute.ClassType(
+                                                                                                                              sourceClassName))
+                                                                                                                  }
+                                                                                                              )
+                                                                                                          }
```

Comment:
Why blank lines here?

Reply:
These are not blank :) Scroll to the right.

Reply:
I could reformat it manually a little.

Reply:
Updated, should be better now.

Reply:
LTGM